utils/plottools.py

```diff
@@ -51,27 +64,23 @@ def _get_cmap(self, cmap_name='jet'):
         return cmap
 
     def _determine_ticks_and_lim(self, ax_name, ax_lim):
-        if type(ax_lim) == type(None):
-            # use the ticks and limit in class setting
-            lim   = (self.DIMS[ax_name].min(), self.DIMS[ax_name].max())
-            ticks = self.DIM_TICKS[ax_name]
+        if ax_name=='t':
+            if type(ax_lim) == type(None):
+                return (self.DIMS[ax_name][0],self.DIMS[ax_name][-1]), None
+            else:
+                return ax_lim, None
```

Comment:
The np.min(), np.max() also work on the np.datetime64, so it should not to independent the function ?

Reply:
You're right. i'm not sure why i through it could be a special case when involving datetime64 object. XD